### PR TITLE
Website/grants update

### PR DESCRIPTION
**Pull request by danielschifano**

### CODE REVIEW

**Comment by kangarang** [comment]
@akuanti yes, i think that's a good idea, since it's not being used or tested. @danielschifano once you transfer these changes over to the `gatsby-site` directory files, feel free to either delete the `/website` directory or leave it as is and i can delete it on the next change.

**Comment by danielschifano** [comment]
> @akuanti yes, i think that's a good idea, since it's not being used or tested. @danielschifano once you transfer these changes over to the `/gatsby-site` directory files, feel free to either delete the `/website` directory or leave it as is and i can delete it on the next change.

Updated :)

**Comment by danielschifano** [comment]
> The changes in the HTML need to be in `gatsby-site/src/pages/grants.js`, since we're no longer using the `website` directory. @kangarang should we delete that directory?

Made the change. Sorry about that.

File: gatsby-site/src/pages/grants.js [hunk 1]
```diff
@@ -77,28 +84,297 @@ const Grants = () => {
...
+                    target="_blank"
+                    className="link dim blue f7 fw7"
+                  >
+                    View website
+                  </a>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section className="w-33-l w-50-m w-100 pa3">
```
Comment (akuanti) [inline gatsby-site/src/pages/grants.js]: This block is a repetition of the previous block.

File: gatsby-site/src/pages/grants.js [hunk 2]
```diff
@@ -77,28 +84,297 @@ const Grants = () => {
...
-          <a href="https://disputes.panvala.com/slates" target="_blank" rel="noopener noreferrer">
-            <button className="f6 link dim bn br-pill pv3 ph4 white bg-teal fw7 pointer">
-              Explore the App
-            </button>
-          </a>
         </section>
       </div> */}
 
+      {/* <!-- Batch 4 --> */}
+      <section className="w-70-l w-80-m w-90 center tc mv6">
```
Comment (akuanti) [inline gatsby-site/src/pages/grants.js]: In the other blocks, the cards all appear to have very similar heights, but for Batch 4, they are all different heights. Is this intentional? Is it just a coincidence?

File: gatsby-site/src/pages/grants.js [hunk 3]
```diff
@@ -77,28 +84,297 @@ const Grants = () => {
...
+                <div>
+                  <h4 className="f6 fw7 ma0">Vyper Security Audit</h4>
+                  <h5 className="f7 fw7 blue ma0 mt3">200,000 PAN</h5>
+                  <p className="ma0 f7 lh-text mt3">
+                    A proposal to make Ethereum safer by completing Vyper audit which is necessary to ensure that contracts behave as expected
+                  </p>
+                </div>
+                <div>
+                  <a href="https://diligence.consensys.net/"
+                    target="_blank" c
```
Comment (akuanti) [inline gatsby-site/src/pages/grants.js]: One more thing: `className` got split onto the next line here.

File: gatsby-site/src/pages/grants.js (hunk 2 above)
Comment (danielschifano) [reply gatsby-site/src/pages/grants.js]: The heights can vary based on how much text is in the card. Batch 4 just happens to have more text than previous batches.

File: gatsby-site/src/pages/grants.js (hunk 3 above)
Comment (danielschifano) [reply gatsby-site/src/pages/grants.js]: Fixed.

File: gatsby-site/src/pages/grants.js (hunk 1 above)
Comment (danielschifano) [reply gatsby-site/src/pages/grants.js]: Fixed

**Pull request merged**